controllers/hardwareclassification_controller.go

```diff
@@ -105,7 +105,11 @@ func (hcReconciler *HardwareClassificationReconciler) Reconcile(req ctrl.Request
 
 	//Extract the hardware details from the baremetal host list
 	validatedHardwareDetails := hcManager.ExtractAndValidateHardwareDetails(extractedProfile, hostList)
-	hcReconciler.Log.Info("Validated Hardware Details From Baremetal Hosts", "Validated Host List", validatedHardwareDetails)
+	hcReconciler.Log.Info("Validated Hardware Details", "HardwareDetails", validatedHardwareDetails)
+
+	//Compare the host list with extracted profile and fetch the valid host names
+	validHosts := hcManager.MinMaxFilter(hardwareClassification.ObjectMeta.Name, validatedHardwareDetails, extractedProfile)
+	hcReconciler.Log.Info("Filtered Bare metal hosts", "ValidHosts", validHosts)
```

Comment:
do I get it correctly that at this point we do not do anything yet with the hosts that match the HWC?

Reply:
@maelk  We will going to need that in the next PR for label update.

Reply:
ok